packages/react-components/src/localization/locales.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import en_US from './translated/en-US.json';

```

Comment:
@prprabhu-ms were there any concerns here about treeshaking? I remember having a disucssion but can't remember the outcome

Reply:
I wonder if dynamic importing would help here https://2ality.com/2017/01/import-operator.html#loading-code-on-demand

Reply:
(@mgamis-msft I wouldn't block the PR on this but we will need to solve this if a consumer can't treeshake the languages they don't want)

Reply:
Thanks for the dynamic import suggestion. I was thinking something similar like async callback prop on `LocalizationProvider` that returns the component strings which can be assigned something like:
`const loadLocaleStrings= async (locale: string): Promise<ComponentStrings> => {
  switch (locale) {
    ...
    case "en-US":
    default:
      return (await import("translated/en-US.json")).default;
  }
};
`
but I will address it in bug https://skype.visualstudio.com/SPOOL/_workitems/edit/2495870. Thanks.